test/__snapshots__/roundtripValidatorTests.ts.snap

```diff
@@ -86,7 +86,7 @@ Object {
     Object {
       "code": "ROUNDTRIP_ADDITIONAL_PROPERTY",
       "jsonPathsInPayload": Array [],
-      "message": "The property is returned in the GET response, but it is not declared in the PUT request.",
+      "message": "The property is returned in the GET response, but it is not declared in the PUT request. Additional value is \\"ReadWrite\\", path: \\"/properties/storageProfile/osDisk/caching\\".",
```

Comment:
just realized the path is already there, so it seems we can skip outputting path in message

Reply:
`The property 'caching' is returned in the GET response, but it is not declared in the PUT request.`